uitools/toolkitcpp/CMakeLists.txt

```diff
@@ -102,6 +102,7 @@ set(esri_arcgisruntime_toolkit_images_resource_files
     "images/organization.svg"
     "images/pause.svg"
     "images/pin-tear.svg"
```

Comment:
Maybe remove SVG unused files?

Reply:
`pin-tear.svg` is still used in callout demo to show the icon and it is working.